reporter/src/main/kotlin/reporters/ExcelReporter.kt

```diff
@@ -76,7 +83,10 @@ class ExcelReporter : TableReporter() {
 
     private lateinit var creationHelper: CreationHelper
 
-    override fun generateReport(tabularScanRecord: TabularScanRecord, outputDir: File): File {
+    override fun generateReport(ortResult: OrtResult, resolutionProvider: ResolutionProvider,
+                                copyrightGarbage: CopyrightGarbage, outputDir: File, postProcessingScript: String?)
```

Comment:
Put the parameters on separate lines:
https://kotlinlang.org/docs/reference/coding-conventions.html#function-formatting